installscripts/jazz-terraform-unix-noinstances/jenkins.tf

```diff
@@ -80,6 +80,9 @@ resource "null_resource" "update_jenkins_configs" {
   provisioner "local-exec" {
     command = "${var.modifyPropertyFile_cmd} ACCOUNTID ${var.jazz_accountid} ${var.jenkinsjsonpropsfile}"
   }
+  provisioner "local-exec" {
+    command = "${var.modifyPropertyFile_cmd} ACCOUNT_ID ${var.jazz_accountid} ${var.jenkinsjsonpropsfile}"
```

Comment:
whats the diff between this and lines 80-82

Reply:
In oss, the installer vars json already have the region and account, but the structure is not like this,

"DEFAULTS": {
"REGION": "$foo",
"ACCOUNT_ID": "$bar"
}

Reply:
Let's fix that then.

Rather than update 2 different variable names, let's bring the internal version into sync with the variable names OSS uses and do it once, rather than introducing a workaround.

Reply:
Agreed, let's just use one variable name. Initial ask was to allow the installer to add in default settings (account, region etc.) into the configuration file. Key `ACCOUNT_ID` mentioned in the requirements was only for guidance, if we are using key `ACCOUNTID` elsewhere, please use the same.

Reply:
Done.